network/src/main/java/javasabr/mqtt/network/message/in/PublishReleaseMqttInMessage.java

```diff
@@ -72,13 +74,11 @@ protected boolean validMessageFlags(byte messageFlags) {
   @Override
   protected void readVariableHeader(MqttConnection connection, ByteBuffer buffer) {
     super.readVariableHeader(connection, buffer);
-
-    // http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718055
-    messageId = readShortUnsigned(buffer);
```

Comment:
Does this change reading flow? Will next read (line 79) have messageId instead of reasonCode?